capability-fd.cc

```diff
@@ -1312,7 +1312,10 @@ TEST(Capability, NoBypassDACIfRoot) {
   pid_t child = fork();
   if (child == 0) {
     // Child: change uid to a lesser being
-    setuid(other_uid);
+    ASSERT_NE(0u, other_uid) << "other_uid not initialized correctly, "
+                                "please pass the -u <uid> flag.";
```

Comment:
This might be better as a single check at start-of-day, maybe around line 51 of capsicum-test-main.cc ?

Reply:
That's what I initially had, but I decided to move it to the two tests that need a non-zero value so that you can run individual tests with --gtest_filter=... without having to pass -u.

I guess I could also try change the FreeBSD build system to install the binary as not owned by root?

Reply:
Fine as-is if you're running tests individually (I always used to run the whole suite).